src/main/java/soot/SootClass.java

```diff
@@ -84,6 +84,8 @@
 
   public final static String INVOKEDYNAMIC_DUMMY_CLASS_NAME = "soot.dummy.InvokeDynamic";
 
+  public final static String JAVA_LANG_OBJECT = "java.lang.Object";
```

Comment:
There is a singleton for the Object type in the Scene which you could use: `Scene.v().getObjectType()`